### PR TITLE
Fix dev task on package.json

### PR DESCRIPTION
**Pull request by daviferreira**
Not sure if this is the right task to run on the dev script or if we could run just the default grunt task. Let me know :smile:

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: app/templates/_package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "grunt dev",
+    "dev": "grunt concurrent:dev",
```
Comment (redonkulus) [inline app/templates/_package.json]: The default Grunt task will run jshint and concurrent:dev so you could remove the `concurrent:dev` part. You are correct that `dev` is not a correct Grunt task.

**Comment by mridgway** [comment]
Hi, sorry for missing this one. Looks like these changes are already in master now. Thanks for the PR!

**Pull request closed without merging**